osquery/tables/system/windows/logon_sessions.cpp

```diff
@@ -0,0 +1,93 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+// clang-format is turned off to ensure windows.h is first
+// clang-format off
+#include <Windows.h>
+// clang-format on
+#include <NTSecAPI.h>
+#include <sddl.h>
+#include <tchar.h>
+#include <vector>
+
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/windows/process_ops.h"
+#include "osquery/filesystem/fileops.h"
+
+#define STATUS_SUCCESS ((NTSTATUS)0x00000000L)
```

Comment:
Hm. So I wasn't able to get this working either, there's definitely something funky happening with our includes, as from what I can tell the redefinition warnings shouldn't be happening, as I can't find anywhere that we're including `winnt.h`, which to me indicates it might be something bootstrapped in from `windows.h`, but overall the `STATUS_SUCCESS` should be defined _already_ without needing another include. Super strange. That being said, I'm really not a fan of `#define`ing something that's defined in one of the Windows headers. As a compromise, how about making a `unsigned long kLsaStatusSuccess = 0;` at the `tables` namespace and checking against that?